### ISSUE TITLE
Why aligntab is not allowed?

### ISSUE THREAD

**Issue by aleixfabra** [cakephp/cakephp-codesniffer#132]
This is cleanest

```php
$data = [
    'word'     => 'hello',
    'longword' => 'hello'
];
```
than this

```php
$data = [
    'word' => 'hello',
    'longword' => 'hello'
];
```

**Comment by ravage84** [comment]
Because this leads to many unnecessary whitespace changes.
This is unwanted, distracting "white noise" in the VCS and has very limited benefits ("looks cleaner", arguably).

Years ago I had the same opinion and some programming languages benefit more than others, I guess.
But most PHP developers think this is a bad habit and I agree.

**Comment by markstory** [comment]
For exactly the reasons @ravage84 outlined.  You can always disable that sniff by creating a custom ruleset that ignores/excludes/disable the tab alignment sniffs.
